tools/pontoon/teams_projects.md

```diff
@@ -0,0 +1,88 @@
+
+![Project hover](/assets/images/pontoon/teams_projects/project_hover.png)
+
+### Requesting a project
+
+It’s possible to request a project from a locale’s *Team page*.
+
+![Request a project](/assets/images/pontoon/teams_projects/request_project.png)
+
+Click on *REQUEST MORE PROJECTS* (highlighted in red), select the project you want to add and then click *REQUEST NEW PROJECT* (you need to select at least one project for the button to be displayed).
+
+![Request a project](/assets/images/pontoon/teams_projects/request_project_selected.png)
+
+An email will be sent to Pontoon’s administrators, and the localization driver in charge of the project will act on the request. Please note that:
+* Some projects have a closed list of supported locales, meaning that you can’t request the project on Pontoon.
+* Some projects can be requested but eventually not enabled, because of practical restrictions related to the project itself (e.g. lack of support for the locale in iOS).
+
+## Project page
+
+You can access a project’s page by selecting it from the [Projects page](https://pontoon.mozilla.org/projects/).
+
+This is a screenshot of the [Mozilla.org project page](https://pontoon.mozilla.org/projects/mozillaorg/):
+
+![Project page for Mozilla.org](/assets/images/pontoon/teams_projects/project_page.png)
+
+The page header contains information about the current project (same as the [Localization page](#localization-page)), plus an indication of the overall status of completion and statistics across all enabled languages.
+
+![Header of project page for Mozilla.org](/assets/images/pontoon/teams_projects/project_page_header.png)
+
+There are 3 tabs available (highlighted in red):
+
+* **Teams**: a list of all locales enabled in Pontoon for this project.
+* **Contributors**: a list of active contributors with their statistics, filterable by time (all time, 1/3/6/12 months).
+* **Info**: information about this project.
+
+## Localization page
+
+You can access a project’s *Localization page* in two ways:
+* By selecting it from the [Team page](#team-page).
+* By selecting it from the [Project page](#project-page).
+
+![Header of localization page](/assets/images/pontoon/teams_projects/localization_page_header.png)
+
+The page header contains useful information about this project:
+* **Priority**: it goes from 5 stars (highest priority) to 1 star (lowest priority).
+* **Deadline**: set only for some projects, it indicates when the translation is due.
+* **Repository**: link to the repository storing translations for this project.
+* **Resources**: links to useful external resources. Custom links can be defined for each project.
+* **Contact person**: localization driver in charge of this project.
```

Comment:
I'd move this seciton under the Project page, because that's where users will see it first (in this document and during Pontoon use).